src/index.tsx

```diff
@@ -12,16 +12,36 @@ import {
   jsonformsReducer,
   RankedTester
 } from '@jsonforms/core';
-import { editorReducer } from '@jsonforms/editor';
+import {
+  editorReducer,
+  findAllContainerProperties,
+  Property,
+  setContainerProperties
+} from '@jsonforms/editor';
 import { uiEditorReducer } from './reducers';
 import { NonEmptyLayoutRenderer, nonEmptyLayoutTester } from './editor/util/NonEmptyLayout';
 import * as JsonRefs from 'json-refs';
+import * as _ from 'lodash';
 
 const uischema = {
   'type': 'MasterDetailLayout',
   'scope': '#'
 };
 
+export const filterPredicate = (data: Object) => {
+  return (property: Property): boolean => {
+    if (!_.isEmpty(modelMapping) &&
+      !_.isEmpty(modelMapping.mapping)) {
+      if (data[modelMapping.attribute]) {
+        return property.schema.id === modelMapping.mapping[_.toLower(data[modelMapping.attribute])];
```

Comment:
Do we still need to use `toLower`?

Reply:
yes, for instance type name is `HorizontalLayout`, and in my mapping configuration file I set the corresponding id to `horizontallayout`

Reply:
Yes, please do that as that is clearer IMHO.